backend/app/views/campaigns/_base.json.jbuilder

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+json.id campaign.id
+json.name campaign.name
+json.description campaign.description
+json.promisedAmount campaign.promised_amount
```

Comment:
Missing couponDenomination

Reply:
Do we need it for the base view? Or should I add just to the admin views

Reply:
I think we can leave it since the frontend types do not have it?